scripts/module/keybinder.nut

```diff
@@ -37,8 +37,28 @@ function onPlayerCommand( player, cmd, text ) {
     }
     bindTable.rawset(key, argTable.rawget("second"));
     binds[player.ID] = bindTable;
+  } else if (cmd == "unbind") {
```

Comment:
If we only allow one bind per key, why not just do `/bind <key>` without a parameter?

Reply:
That's a very good idea!